sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/administration/ServiceBusAdministrationClientBuilder.java

```diff
@@ -321,7 +338,10 @@ private HttpPipeline createPipeline() {
         final String clientName = properties.getOrDefault("name", "UnknownName");
         final String clientVersion = properties.getOrDefault("version", "UnknownVersion");
 
-        httpPolicies.add(new UserAgentPolicy(httpLogOptions.getApplicationId(), clientName, clientVersion,
+        final String applicationId = CoreUtils.validateApplicationId(httpLogOptions, clientOptions,
+            new IllegalStateException("applicationId should be same in httpLogOptions and clientOptions."));
```

Comment:
I don't think we need to do this. We should deprecate the HttpLogOptions one, and have it take lower precedence to the clientOptions one (so we only use it when the clientOptions one is not set).